js/coinex.js

```diff
@@ -3092,6 +3093,89 @@ module.exports = class coinex extends Exchange {
         };
     }
 
+    parseTransfer (transfer, currency = undefined) {
+        //
+        // fetchTransfers
+        //
+        //     {
+        //         "amount": "10",
+        //         "asset": "USDT",
+        //         "transfer_type": "transfer_out", // from swap to spot
+        //         "created_at": 1651633422
+        //     },
+        //
+        const timestamp = this.safeTimestamp (transfer, 'created_at');
+        const transferType = this.safeString (transfer, 'transfer_type');
+        let fromAccount = undefined;
+        let toAccount = undefined;
+        if (transferType === 'transfer_out') {
+            fromAccount = 'swap';
+            toAccount = 'spot';
+        } else if (transferType === 'transfer_in') {
+            fromAccount = 'spot';
+            toAccount = 'swap';
+        }
+        if (currency === undefined) {
+            currency = this.safeString (transfer, 'asset');
+        }
+        return {
+            'id': undefined,
+            'timestamp': timestamp,
+            'datetime': this.iso8601 (timestamp),
+            'currency': currency,
+            'amount': this.safeString (transfer, 'amount'),
```

Comment:
We have to use this.safeNumber here, since this returned structure is not wrapped with another post-parsing call, can't return plain strings here.